manifests/common/ceilometer.pp

```diff
@@ -3,10 +3,12 @@
 class openstack::common::ceilometer {
   $is_controller = $::openstack::profile::base::is_controller
 
+  $ceilometer_management_address = $::openstack::config::ceilometer_address_management
   $controller_management_address = $::openstack::config::controller_address_management
 
+  $mongo_username = $::openstack::config::ceilometer_mongo_username
   $mongo_password = $::openstack::config::ceilometer_mongo_password
-  $mongo_connection = "mongodb://${controller_management_address}:27017/ceilometer"
+  $mongo_connection = "mongodb://${mongo_username}:${mongo_password}@${ceilometer_management_address}:27017/ceilometer"
```

Comment:
What happens if the user hasn't defined $mongo_username and $mongo_password? Would it make sense to wrap this in `if $mongo_username and $mongo_password` and allow the old mongo connection if they haven't defined the user?